libs/blocks/preflight/checks/asoApi.js

```diff
@@ -20,7 +20,7 @@ export const asoCache = {
 
 const lanaLog = (message) => {
   window.lana.log(message, {
-    sampleRate: 1,
+    sampleRate: 100,
```

Comment:
And I'm guessing this is assumed to be safe because it'd be run by authors, so not a lot of traffic? Or because we actually want all errors? Would this be the case if these same checks were run from Milo Studio?